### ISSUE TITLE
Add ? next to search bar and link to search types help section

### ISSUE THREAD

**Issue by ManjotS** [KhalisFoundation/sttm-web#130]
Include tooltip from foundation (http://foundation.zurb.com/sites/docs/tooltip.html)

that appears for each search type and changes when search type changes.

For example for the romanized search, old site had text:

> English Search
> 
> Welcome to the new English Search facility which lets you type in plain english to search Gurbani. Enter the words as you would say them in English. The words should be from the start of the Shabad line.
> 
> For example 'Jo Maangay Taakur Apnay Thay'.
> 
> For optimum results you need to enter at least four words. Five will almost guarantee a find on the first page of results if the words are in the right order.
> 
> Hint:
> If the Shabad line you are searching for is the main line the Keertanee is singing, they may not be singing it from the start of the line. For instance the Shabad "Mere Raam Rai Thoo Santhaa Kaa Santh Theray" may be sang from "Thoo Santhaa Kaa Santh Theray". The easiest thing to do is wait for the next line to be sang then use that one. In this case that would be "Theray Sevak Ko Bhou Kich Nahee...". 

(we would probably want to reduce the amount of text to be more concise)

**Comment by bogas04** [comment]
Now that we have help page, do we need this? We can probably add `?` next to search bar and link it to correct help section.

**Comment by ManjotS** [comment]
agree.. should copy the old help section into new one and then link to it

**Comment by bogas04** [comment]
@ManjotS @tarunsingh5

**Comment by ManjotS** [comment]
I dont see a way to get to help easily from the search bar

**Comment by bogas04** [comment]
Oh yeah. Will take it up soon

**Comment by bogas04** [comment]
Or if anyone else wants to take it, feel free!

**Comment by saintsoldierx** [comment]
Closing this as we have a better help section now.

**Comment by saintsoldierx** [comment]
![image](https://user-images.githubusercontent.com/27791330/115136788-ecd2fc80-9fd6-11eb-9bf3-69feab1c1b32.png)

Add icon next to search.
Opens new tab/window
Point to https://support.khalisfoundation.org/support/solutions/articles/63000076434-what-do-the-different-search-types-do-

**Comment by tsingh777** [comment]
Support page looks a little out of date. might want a separate ticket for that?

**Comment by gsin11** [comment]
I was checking the feasibility of adding this link next to the search bar and found this:
![image](https://user-images.githubusercontent.com/3865313/115157268-afc74280-a088-11eb-9737-47574eb39617.png)
I just added a "?" there and you can see search is already getting shorter in width. With proper styling, the search field will become shorter in width.
Are we fine to have such behaviour for Mobile users?

**Comment by saintsoldierx** [comment]
I would still like to add a ? icon as per my screenshot above with it going to support page. I am fine with the search bar becoming shorter in width for mobile devices.

**Comment by saintsoldierx** [comment]
Link should go to: https://support.khalisfoundation.org/support/solutions

**Comment by gurjit03** [comment]
Implemented already
